File: src/codes/Users_Manual/advanced/ultrasound_freq.rs
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() {
Controller::builder([AUTD3::new(Vector3::zeros())]).with_ultrasound_freq(41 * kHz)
# .open(autd3::link::Nop::builder());
# }

File: src/codes/Users_Manual/advanced/ultrasound_freq.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Utils;
~using AUTD3Sharp.NativeMethods;
~using static AUTD3Sharp.Units;
~var autd =
Controller.Builder([]).WithUltrasoundFreq(41 * kHz)
~;